v2/internal/pkg/release/cincinnati.go

```diff
@@ -95,6 +95,7 @@ type CincinnatiParams struct {
 	Arch         string
 }
 
+// nolint
```

Comment:
Please, let's not do this. The commit says "Fix linting errors" but they are being ignored instead.
If we are going to ignore a lint error, let's do it for a specific linter with an explanation of why it's being ignored.
In this case, all you need to fix the linter is to change the return type to `CincinnatiSchema`